tests/EdFi.Security.DataAccess.UnitTests/EdFi.Security.DataAccess.UnitTests.csproj

```diff
@@ -18,7 +18,7 @@
   <ItemGroup>
 	  <PackageReference Include="FakeItEasy" Version="8.1.0" />
 	  <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
-	  <PackageReference Include="NUnit" Version="4.0.1" />
+	  <PackageReference Include="NUnit" Version="3.13.3" />
```

Comment:
Please switch back to NUnit 4 unless there is some reason we cannot use it. If we cannot use it, then please pin the version so that it is impossible to update it, for example with `Version="[3.13.0,4.0.0)"`.